ctapipe/io/eventfilereader.py

```diff
@@ -60,76 +97,21 @@ def __init__(self, config, tool, **kwargs):
         """
         super().__init__(config=config, parent=tool, **kwargs)
 
-        if self.origin is None:
-            raise ValueError("Subclass of EventFileReader should specify "
-                             "an origin")
-
-        self._num_events = None
-        self._event_id_list = []
-
-        if self.input_path is None:
-            raise ValueError("Please specify an input_path for event file")
-        self._init_path(self.input_path)
+        self._metadata = dict(is_simulation=False)
```

Comment:
Can this class really already know if `is_simulation` is True or False?

Reply:
At least at the moment only outputs from sim_telarray are stored in hessio files